pkg/operation/common/types.go

```diff
@@ -132,6 +132,9 @@ const (
 	// KubeAPIServerDeploymentName is the name of the kube-apiserver deployment.
 	KubeAPIServerDeploymentName = "kube-apiserver"
 
+	// EnableHPANodeCount is the number of nodes in shoot cluster after which HPA is deployed to autoscale kube-apiserver.
+	EnableHPANodeCount = 5
```

Comment:
Why do we enable the HPA for Shoots with >= 5 nodes? Isn't that value quite low? Shouldn't we only enable it for larger clusters?

Reply:
From the data that I have collected, I have seen some improvement in the responsiveness and stability of the apiserver under load for nodes = 10. Taking a more conservative approach, and after collecting some more data, I decided to use 5 as threshold to deploy/activate HPA. This will also give us a chance to observe apiserver's behaviour in the actual deployments, using which we can fine tune our values further.

Reply:
OK, agreed.